### ISSUE TITLE
Invest time looking into moving sidebar to the top

### ISSUE THREAD

**Issue by daattali** [daattali/oldschool-github-extension#20]
This is ambitious, but a lot of people emailed me asking for this. Perhaps it's possible.

**Comment by zekefarwell** [comment]
I've messed around with it a little and it's definitely possible with only css by changing the `flex-direction` from `row` to `column-reverse`.  But then there's a lot more to do after that!  Lots of little tweaking and hiding things needed.  Pretty tough to get it exactly as it was before, but something semi-decent should be possible.

Looks like this fork by @sreenivasanramesh has a working method, but the css breaks other pages at the moment:
https://github.com/sreenivasanramesh/old-github-ui
https://www.reddit.com/r/github/comments/hh2qro/moved_the_sidebar_to_the_top_extensioncss_in/

**Comment by daattali** [comment]
Getting it close to before but not exactly is fine, as long as it looks good and doesn't have too many eough edges. I'll look at the fork

**Comment by daattali** [comment]
I just tried the fork, and it looks like it's a decent first step, but there are indeed some things that have to be patched. Would be cool if all the kinks are able to be worked out and it can be applied to the entire site

**Comment by sreenivasanramesh** [comment]
Mine is a hack for sure. I'm not applying my CSS on certain pages like the issues page, etc.
Github's AJAX is breaking the CSS when I'm navigating from the pages where I'm not applying the CSS to somewhere where I expect the CSS to be present. I'm not sure how to take care of that.

**Comment by daattali** [comment]
Yeah github's AJAX thing is annoying. It's the only reason I went full CSS and not javascript.

**Comment by sreenivasanramesh** [comment]
I was able to get the CSS working for most pages with some CSS selectors.  
A couple of pages like https://github.com/*/*/community and https://github.com/*/*/settings/interaction_limits are breaking.  Other than that, there are a couple of places where the CSS needs to be polished.

I'll send a PR in a while.

**Comment by daattali** [comment]
Awesome! When I looked yesterday, I noticed that on the repo page, the README was wider than the file list above it, and the section for "Contributors"/"Packages"/"Releases"/"Environments"/etc was looking not so great when the screen was made smaller. Were those issues worked out?

**Comment by sreenivasanramesh** [comment]
Yeah, I've got to fix those issues.  
Both the README and the language bar are a bit wider than the files list, but should be easy to fix.  


The "Contributors"/"Packages"/"Releases"/"Sponsor"/"Environments"/etc section is somewhat tricky to fix. I've never worked with CSS before this, so I'm still trying to figure out how to make that work.

**Comment by daattali** [comment]
This is not the easiest way to get introduced to CSS, good job doing all this!

**Comment by acecilia** [comment]
Would love to see this happening, the not-centered readme is extremely annoying. Good job! thanks :)

**Comment by sreenivasanramesh** [comment]
I've created a draft PR.  
I'm blocked on fixing the Contributors/Packages/Releases/Sponsor... section. I'm not sure how to do that.

**Comment by daattali** [comment]
In https://github.com/daattali/oldschool-github-extension/issues/14#issuecomment-651779984 , @chocolateboy showed that it's possible and not too ugly to use some JS. So it might be worthwhile to do this sidebar translation with JS instead of CSS
